Source/NujasWidget/Public/NujasWidget.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018-2019 The Janus Project | 2034 Complex LLC. All Rights Reserved.
+
+#pragma once
+
+#include "UserWidget.h"
```

Comment:
Hmm, I would recommend using Blueprint/UserWidget.h like in the doc:
https://docs.unrealengine.com/en-US/API/Runtime/UMG/Blueprint/UUserWidget/index.html

Just to give it a bit more clarity that this is a BP class.